orix/quaternion/misorientation.py

```diff
@@ -263,6 +264,37 @@ def from_scipy_rotation(
             M.symmetry = symmetry
         return M
 
+    @classmethod
+    def from_path_ends(
+        cls, points: Misorientation, closed: bool = False, steps: int = 100
+    ) -> Misorientation:
+        """Return (mis)orientations tracing the shortest path between
+        two or more consecutive points.
+
+        Parameters
+        ----------
+        points
+            Two or more (mis)orientations that define waypoints along
+            a path through rotation space (SO3).
+        closed
+            Option to add a final trip from the last waypoint back to
+            the first, thus closing the loop. The default is False.
+        steps
+            Number of (mis)orientations to return along the path
+            between each pair of waypoints. The default is 100.
+
+        Returns
+        -------
+        path
+            quaternions that map a path between the given waypoints.
+        """
+        out = super().from_path_ends(points=points, closed=closed, steps=steps)
+        path = cls(out.data)
+        # copy the symmetry if it exists.
+        if hasattr(points, "_symmetry"):
+            path._symmetry = points._symmetry
```

Comment:
What I mean above (https://github.com/pyxem/orix/pull/558/files#r2215490352) is that we should drop the check and just do this:

```suggestion
        path._symmetry = points._symmetry
```

If not (mis)orientations are given, this will raise an attribute error. Users should see this error to learn that they should instead use the equivalent rotation or quaternion method, instead of the call passing silently.

If we want to be more kind to the user, and perhaps save unnecessary computation, we can make a check at the top for type:

```python
if not isinstance(points, Misorientation):
    raise TypeError(f"Points must be a (mis)orientation, not of type {type(points)}")
```

After this is 100% safe to access the symmetry.